content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -362,6 +362,42 @@ For more information on inter-pod affinity/anti-affinity, see the
 You may want to check [Taints](/docs/concepts/configuration/taint-and-toleration/)
 as well, which allow a *node* to *repel* a set of pods.
 
+## nodeName
+
+`nodeName` is the simplest form of node selection constraint, but due
+to its limitations its use is _not_ recommended.  `nodeName` is a
+field of PodSpec.  If it is non-empty, the scheduler ignores the pod
+and the kubelet running on the named node tries to run the pod.  Thus,
+if `nodeName` is provided in the PodSpec, it takes precedence over the
+above methods for node selection since satisfying their constraints is
+implemented in the scheduler.
+
+Some of the limitations of using `nodeName` to select nodes are:
+
+-   If the named node does not exist, the pod will be deleted after a
+    time pending.
+-   If the named node does not have the resources to accommodate the
+    pod, the pod will fail and its reason will indicate why,
+    e.g. OutOfmemory or OutOfcpu.
+-   Node names in cloud environments are not always predictable or
+    stable.
+
+Here is an example of a pod config file using the `nodeName` field:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+  nodeName: kube-01
+```
+
+The above pod will be scheduled onto the node kube-01.
```

Comment:
nit: since scheduler is not involved in placing the pod on the node, I would not use "scheduled". You can say `The above pod will run on the node kube-01.`.

Reply:
Changed